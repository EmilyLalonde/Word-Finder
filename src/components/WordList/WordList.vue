<template>
  <div id="wordList">
    <ul>
      <li
        v-for="(synonym, index) in this.synonyms"
        v-bind:key="index"
        @click.prevent="findWords(synonym)">
        {{synonym}}
        </li>
    </ul>
  </div>
</template>

<script>
export default {
  name: 'list',
  props: ['synonyms', 'findWords']
};
</script>

<style scoped>
ul {
  list-style-type: none;
}
</style>