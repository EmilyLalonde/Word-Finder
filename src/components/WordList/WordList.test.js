import { mount } from '@vue/test-utils';
import WordList from './WordList.vue';

describe('WordList', () => {

  test('renders correctly', () => {
    const wrapper = mount(WordList);
    expect(wrapper.element).toMatchSnapshot();
  })
})