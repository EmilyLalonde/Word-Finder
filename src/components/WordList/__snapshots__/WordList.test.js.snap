// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordList renders correctly 1`] = `
<div
  id="wordList"
>
  <ul />
</div>
`;
